### PR TITLE
Fixed #26528 -- Allowed any iterable (e.g. tuple) as validators kwarg for form/model fields.

### PR DESCRIPTION
**Pull request by loic**
https://code.djangoproject.com/ticket/26528

### CODE REVIEW

File: tests/forms_tests/tests/test_validators.py [hunk 1]
```diff
@@ -8,38 +8,36 @@
 from django.core.exceptions import ValidationError
 
 
-class UserForm(forms.Form):
```
Comment (timgraham) [inline tests/forms_tests/tests/test_validators.py]: I'd like if you made this change in a separate commit so it doesn't confuse this one.

**Comment by timgraham** [comment]
LGTM. Maybe you'd like to fix those flake8 warnings and use past tense in the commit message.

File: tests/forms_tests/tests/test_validators.py (hunk 1 above)
Comment (loic) [inline tests/forms_tests/tests/test_validators.py]: Didn't realize the diff would look so confusing, I just indented the form. Will move to another commit.
